plugin/plaintextmodel.cpp

```diff
@@ -264,6 +273,25 @@ bool PlainTextModel::readLines(
     return false;
 }
 
+QTextCodec *PlainTextModel::detect(const char *encodedCharacters, int length)
```

Comment:
Detect is slightly vague as name, something like detectCodec might be better.

Reply:
Done

Reply:
Done